memory/ndctl.py

```diff
@@ -398,6 +413,24 @@ def test_multiple_ns_multiple_region(self):
                     region=region, mode=self.mode_to_use, size=self.size)
                 self.log.info("Namespace %s created", nid + 1)
 
+    def test_nslot_namespace(self):
+        """
+        Test max namespace with nslot value
+        """
+        region = self.get_default_region()
+        self.log.info("Using %s for max namespace creation", region)
+        self.disable_namespace()
+        self.destroy_namespace()
+        region_size = self.get_json_val(self.get_json(
+            '-r %s' % region)[0], 'size')
+        namespace_size = 1 << 30
+        count = region_size // namespace_size - 1
+        count = min(count, self.get_slot_count(region) - 1)
```

Comment:
suggestion: instead of getting a `min(count - 1, get_slot_count - 1)`, can't we have `min(count, get_slot_count) - 1` ?